test/e2e/fixture/fixture.go

```diff
@@ -53,7 +54,16 @@ var (
 	apiServerAddress string
 	token            string
 	plainText        bool
-	repoUrl          string
+)
+
+type RepoURLType string
+
+const (
+	RepoURLTypeFile  = "file"
```

Comment:
sometimes constants make code harder to understand - I think you should just use the strings here